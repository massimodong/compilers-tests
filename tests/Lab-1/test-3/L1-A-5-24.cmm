struct hBciT wTg5dEEi ( int MiqLcwQhv0HMi , int JNHxAIf8bPUP 
  ) {
  int _9mAt = ;
  float XP_0o3fuwi5 
  if ( kC1mjFz )   YDcOr2Sjr3fY ~;
else   if ZihG8lLe )   i7JyTPOBSOmO 


}
float #;
``