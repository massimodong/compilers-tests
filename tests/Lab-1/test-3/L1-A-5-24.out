Error Type B at Line 3
Error Type B at Line 5
Error Type A at Line 5
Error Type B at Line 6
Error Type B at Line 9
Error Type A at Line 10
Error Type A at Line 11
Error Type A at Line 11
